apis/management.cattle.io/v3/rke_types.go

```diff
@@ -194,6 +196,8 @@ type KubeAPIService struct {
 	ServiceNodePortRange string `yaml:"service_node_port_range" json:"serviceNodePortRange,omitempty" norman:"default=30000-32767"`
 	// Enabled/Disable PodSecurityPolicy
 	PodSecurityPolicy bool `yaml:"pod_security_policy" json:"podSecurityPolicy,omitempty"`
+	// Enable/Disable AlwaysPullImages admissions plugin
+	AlwaysPullImages bool `yaml:"always_pull_images,omitempty" json:"always_pull_images,omitempty"`
```

Comment:
I believe you can override the default list of admission controllers with the external arguments list. If you feel we have to include this plugin to the list of the default ones, we should just add it. And document on how to disable it.

Reply:
In order to override the list you need to provide the whole list. This isn't easy to determine what's supposed to be in this list, nor convenient to set that option.  The AlwaysPullImages admission controller is recommended as a CIS Best practice, but I believe the downside is you'll end up not taking advantage of docker image cache on the host.

Reply:
@jgreat the downside of making any parameter strongly typed is that:

1)  Exposing every k8s service param can be cumbersome as k8s has way too many
2)  It is not clear if this parameter would take precedence over custom admission controllers passed via external args.

I don't particularly mind for this particular parameter to be exposed as strongly typed though, but in the end it is your call.